specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -579,6 +636,16 @@
             "required": true,
             "type": "string",
             "description": "Client API version."
+        },
+        "HanaInstancePatchTagsParameter": {
```

Comment:
nit: shorter name, like TagsParameter - it should be generic enough if we need to use this same parameter in other APIs
And like SubscriptionIdParameter, it's not specific to HanaInstances as a concept.